src/models/FormFields.ts

```diff
@@ -36,7 +37,8 @@ export abstract class FormField {
   protected constructor(
     public type: string,
     public name: string,
-    public label: string
+    public label: string,
+    public label_two?: string
```

Comment:
otrais label? nevar izdalīt uz to, kam kurš label ir?

Reply:
Jau fixed, drīz būs jauns push, laboju pēdējos test errors!